onadata/apps/api/viewsets/user_profile_viewset.py

```diff
@@ -45,6 +48,9 @@
     UserProfileSerializer
 
 BaseViewset = get_baseviewset_class()  # pylint: disable=invalid-name
+Lockout_time = getattr(settings, 'LOCKOUT_TIME', 1800)
+Max_change_password_attempts = getattr(
```

Comment:
The naming conventions for this would be all in caps. i.e. `LOCKOUT_TIME = getattr(settings, 'LOCKOUT_TIME', 1800)`